src/GovUk.Education.ExploreEducationStatistics.Publisher/Startup.cs

```diff
@@ -107,8 +107,10 @@ public override void Configure(IFunctionsHostBuilder builder)
                 .AddScoped<IReleaseSubjectRepository, ReleaseSubjectRepository>(provider =>
                     new ReleaseSubjectRepository(
                         statisticsDbContext: provider.GetService<PublicStatisticsDbContext>(),
-                        footnoteRepository: provider.GetService<IFootnoteRepository>()))
+                        footnoteRepository: new FootnoteRepository(provider.GetService<PublicStatisticsDbContext>())
```

Comment:
I guess this is where you started to get frustrated with the public ad private db contexts then :)

Reply:
Indeed :smile:, but already been feeling that pain with the ticket for this. It's way too error prone!